service-broker/src/main/java/io/enmasse/osb/api/provision/OSBProvisioningService.java

```diff
@@ -72,6 +72,7 @@ public Response provisionService(@Context SecurityContext securityContext,
         }
 
         String name = request.getParameter("name").orElse(service.getName() + "-" + shortenUuid(instanceId));
+        String namespace = request.getParameter("namespace").orElse(service.getName() + "-" + shortenUuid(instanceId));
```

Comment:
Why default the namespace name?  Wouldn't just rejecting the empty service be better?

Reply:
Agreed, fixed this in latest commit